seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -1891,16 +1891,17 @@
       "releasedOn": "January 1, 2016",
       "head": [
         "var logOutput = \"\";",
-        "var oldLog = console.log;",
+        "var originalConsole = console",
         "function capture() {",
+        "    var oldLog = console.log;",
         "    console.log = function (message) {",
         "        logOutput = message + ''.trim();",
-        "        oldLog.apply(console, arguments);",
+        "        oldLog.apply(originalConsole, arguments);",
```

Comment:
Why not use `originalConsole.log.apply(originalConsole, arguments`?  Wouldn't that ensure that we never have the possibility of using an overwritten version?